packages/apps/kubernetes/templates/helmreleases/cilium.yaml

```
@@ -5,6 +5,10 @@ cilium:
   routingMode: tunnel
   enableIPv4Masquerade: true
   ipv4NativeRoutingCIDR: ""
+  gatewayAPI:
+    enabled: true
+  envoy:
+    enabled: true
```

Let's make this configurable:
```suggestion
  {{- if $.Values.addons.gatewayAPI.enabled }}
  gatewayAPI:
    enabled: true
  envoy:
    enabled: true
  {{- end }}
```